notifier.go

```diff
@@ -195,6 +195,9 @@ func NewNotifierWithOptions(opt *NotifierOptions) *Notifier {
 	n.AddFilter(gopathFilter)
 	n.AddFilter(gitFilter)
 
+	msg := "Unsolicited response received on idle HTTP channel starting with"
+	n.AddFilter(msgHasPrefixFilter(msg))
```

Comment:
create filter `httpUnsolicitedResponseFilter` and use it as `n.AddFilter(httpUnsolicitedResponseFilter)`